docs/DafnyRef/Types.md

```diff
@@ -3720,10 +3720,6 @@ Also, `<` is underspecified. With the above code, one can prove neither `z < x`
 
 ## 19.2. Co-inductive datatypes {#sec-co-inductive-datatypes}
 
-TODO: This section and particularly the subsections need rewriting using
-the least and greatest terminology, and to make the text fit better into
-the overall reference manual.
-
 Whereas Dafny insists that there is a way to construct every inductive
 datatype value from the ground up, Dafny also supports
 _co-inductive datatypes_, whose constructors are evaluated lazily, and
```

Comment:
I suggest removing the `-` from "co-inductive" and "co-induction" (see Wikipedia, which writes "coinduction" without a hyphen).